docs/examples/k6.md

```diff
@@ -0,0 +1,47 @@
+# k6
```

Comment:
This file is possibly a leftover of a previous run as an example, instead of a module.

Let's remove it :)